README.md

```diff
@@ -79,70 +94,69 @@ and build the project:
 $ git clone git@github.com:simplex-chat/simplex-chat.git
 $ cd simplex-chat
 $ stack install
-$ dog-food
 ```
 
-## Running the chat client
+## Usage
+
+### Running the chat client
+
+To start the chat client, run `dog-food` from the terminal (as in [eating your own dog food](https://en.wikipedia.org/wiki/Eating_your_own_dog_food)).
 
-Run `dog-food` (as in "eating your own dog food"), or the downloaded file (if you did not move it to bin folder), to start the chat client.
+By default, app data directory is created in the home directory (`~/.simplex`, or `%APPDATA%/simplex` on Windows), and SQLite database file `smp-chat.db` is initialized in it.
 
-By default, app data directory is created in the home directory (`~/.simplex`, or `%APPDATA%/simplex` on Windows), and SQLite database file `smp-chat.db` is initialized in it. The default SMP server is `smp1.simplex.im:5223#pLdiGvm0jD1CMblnov6Edd/391OrYsShw+RgdfR0ChA=` (base-64 encoded string after server port is the transport key digest) - it is pre-configured in the app.
+The default SMP server is `smp1.simplex.im#pLdiGvm0jD1CMblnov6Edd/391OrYsShw+RgdfR0ChA=` (base-64 encoded string after server host is the transport key digest) - it is pre-configured in the app.
 
 To specify a different file path for the chat database use `-d` command line option:
 
 ```shell
 $ dog-food -d my-chat.db
 ```
 
-If you deployed your own SMP server you can set client to use it via `-s` option:
+If you deployed your own SMP server(s) you can set the client to use it via `-s` option:
 
 ```shell
 $ dog-food -s smp.example.com:5223#KXNE1m2E1m0lm92WGKet9CL6+lO742Vy5G6nsrkvgs8=
 ```
 
 The base-64 encoded string in server address is the digest of RSA transport handshake key that the server will generate on the first run and output its digest.
 
-You can still talk to people using default or any other server, it only affects the location of the message queue when you initiate the connection (and the reply queue can be on another server, as set by the other party's client).
+You can still talk to people using default or any other server; it only affects the location of the message queue when you initiate the connection (and the reply queue can be on another server, as set by the other party's client).
```

Comment:
I'd suggest to revert to a comma - the thoughts in are very interconnected.. Maybe even a dash - the second sentence basically elaborates on the first.
```You can still talk to people using default or any other server - it only affects the location...```